pkg/scheduler/apis/config/v1alpha1/conversion.go

```diff
@@ -31,14 +33,33 @@ func Convert_v1alpha1_KubeSchedulerConfiguration_To_config_KubeSchedulerConfigur
 	if err := autoConvert_v1alpha1_KubeSchedulerConfiguration_To_config_KubeSchedulerConfiguration(in, out, s); err != nil {
 		return err
 	}
+	var profile config.KubeSchedulerProfile
+	if err := v1.Convert_Pointer_string_To_string(&in.SchedulerName, &profile.SchedulerName, s); err != nil {
+		return err
+	}
+	if in.Plugins != nil {
+		profile.Plugins = &config.Plugins{}
+		if err := Convert_v1alpha1_Plugins_To_config_Plugins(in.Plugins, profile.Plugins, s); err != nil {
+			return err
+		}
+	} else {
+		profile.Plugins = nil
+	}
```

Comment:
It's already nil by default. So we can remove the else block.

Reply:
This is pretty much boilerplate code. I think the intention is that if you use a dirty `out` object, it still gets setup properly.